src/enumeratedvalue.rs

```diff
@@ -0,0 +1,115 @@
+extern crate xmltree;
+
+use std::collections::HashMap;
+
+use xmltree::Element;
+use ElementExt;
+
+use helpers::*;
+use parse;
+
+macro_rules! try {
+    ($e:expr) => {
+        $e.expect(concat!(file!(), ":", line!(), " ", stringify!($e)))
+    }
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct EnumeratedValue {
+    pub name: String,
+    pub description: Option<String>,
+    pub value: Option<u32>,
+    pub is_default: Option<bool>,
+    // Reserve the right to add more fields to this struct
+    pub _extensible: (),
```

Comment:
Ahh, yeah. I couldn't work out how to initialise an object with private fields / didn't find anything with wild googling. 

The best solution I came up with was to create `new` constructors for each object, is that the best way to keep it hidden?